helpers/helper.go

```diff
@@ -0,0 +1,14 @@
+package helper
+
+// SortSlice is a generic sort function which uses insertion sort
+// to perform ascending and descending sort on a slice of int or string
+func SortSlice[T int | string](s []T, reverse bool) []T {
+	// insertion sort
+	for i := 1; i < len(s); i++ {
+		for j := i; (!reverse && j > 0 && s[j-1] > s[j]) || (reverse && j > 0 && s[j-1] < s[j]); j-- {
```

Comment:
Is it the best and simpliest way to do the thing? Will it work with unicode strings?

Reply:
I didn't want to use the "sort" library and hence used this. Should I replace this with the sort lib based solution?